.github/workflows/tests.yml

```diff
@@ -21,6 +21,10 @@ jobs:
           FILES: |
             go.mod
             go.sum
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libzmq3-dev pkg-config
```

Comment:
is there a pure go version of zmq?

Reply:
Found https://github.com/go-zeromq/zmq4 

NOt sure if it'll be compatible.

Reply:
Please test and let's see.